tests/unit/acl.tcl

```diff
@@ -113,6 +113,22 @@ start_server {tags {"acl"}} {
         set e
     } {*NOPERM*channel*}
 
+    test {In transaction queue publish/subscribe/psubscribe to unauthorized channel will fail} {
+        r ACL setuser psuser +multi +discard
```

Comment:
```suggestion
        r ACL setuser psuser resetchannels +multi +discard
```

Do need to add  ```resetchannels``` to make the test more explicit?

Reply:
I agree this would make it more robust, but not something I would block a merge for.